src/Compilers/Core/CodeAnalysisTest/EditorConfigTests.cs

```diff
@@ -582,7 +588,120 @@ public void WindowsRootConfig()
             {
                 CreateImmutableDictionary(
                     ("cs000", ReportDiagnostic.Suppress))
-            }, options);
+            }, treeOptions);
+        }
+
+        private void VerifyAnalyzerOptions(
+            (string key, string val)[][] expected,
+            (ImmutableArray<ImmutableDictionary<string, ReportDiagnostic>>,
+             ImmutableArray<ImmutableDictionary<string, string>> analyzerOptions) options)
+        {
+            var analyzerOptions = options.analyzerOptions;
+            Assert.Equal(expected.Length, analyzerOptions.Length);
+
+            for (int i = 0; i < expected.Length; i++)
+            {
+                if (expected[i] is null)
+                {
+                    Assert.Null(analyzerOptions[i]);
+                }
+                else
+                {
+                    AssertEx.SetEqual(
+                        analyzerOptions[i],
```

Comment:
>analyzerOptions [](start = 24, length = 15)

Expected comes first.

Reply:
(As I don't believe you're using the `params` overload here)

---
In reply to: [204575049](https://github.com/dotnet/roslyn/pull/27397#discussion_r204575049) [](ancestors = 204575049)